embedchain/memory/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict
+
+
+def merge_metadata_dict(left: Dict[str, Any], right: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I am wondering if we need the merging of metadata. Can you please elaborate more on this?

Reply:
This is an extra feature to save metadata of the chat message into the table. We are not using it as of now. But if someone wants to store some metadata along with each conversation, then this will enable that.